01_process_occurrences.Rmd

```diff
@@ -0,0 +1,112 @@
+# Workflow to process species occurrences
+
+## Set Up
+
+```{r}
+source("setup.R")
+```
+
+## Explore and Clean Survey Data
+
+Read in raw tables from Access database (PC version)
+
+```{r}
+# NOTE: THIS ONLY WORKS AS IS ON A PC, will likely throw errors on a Mac
```

Comment:
For what it's worth, I do think you can use a hosted desktop to connect a mac to access. But it's clunky and probably easier to just find a PC to run this on in most cases.